bin/plugin/commands/packages.js

```diff
@@ -599,7 +588,7 @@ async function runPackagesRelease( config, customMessages ) {
 			await Promise.all(
 				temporaryFolders
 					.filter( ( tempDir ) => fs.existsSync( tempDir ) )
-					.map( ( tempDir ) => rimraf( tempDir, rethrow ) )
+					.map( ( tempDir ) => rimraf( tempDir ) )
```

Comment:
I'm not entirely sure if this is exactly the same handling, so I would appreciate some verification. The updated version of `rimraf` is Promise-based now, so it seems correct to me.

Reply:
I don't see why `Promise.all` was there before, but it makes sense now if there are promises being used.